clients/cli/cmd/internal/push_source.go

```diff
@@ -148,12 +148,12 @@ func (source *Source) CheckPreconditions() error {
 	return nil
 }
 
-func (sources Sources) ProjectIds() []string {
-	projectIds := []string{}
+func (sources Sources) ProjectIdsBranches() []LocaleCacheKey {
```

Comment:
This function name confused me a bit because it returns a `LocaleCacheKey`. Maybe we could either rename the function or the struct name so that it's more clear what it returns?